### PR TITLE
build(docker): #57 Build from recent image

### PR DESCRIPTION
**Pull request by AlexAxthelm**
Build from a recent image from the public registry for `workflow.transition.monitor`. While these images don't include the pacta-data needed to prepare indices, the current README instructions include a requirement to externally mount PACTA-data.

Closes: #57

### CODE REVIEW

**Comment by AlexAxthelm** [comment]
long-term goal: use explicit pinning on SHA for stability

```
FROM ghcr.io/rmi-pacta/workflow.transition.monitor@sha256:4cb557ea3155f53a16253e85ed5a7986076c58eee335d685dd30183a038ef5b7
```

**Comment by cjyetman** [comment]
maybe should update the README along with this?
https://github.com/RMI-PACTA/workflow.prepare.pacta.indices/blob/cac36eeab6a570de64889c7390fe0a65b00cfcb7/README.md?plain=1#L15-L20

**Review by jdhoffa — CHANGES_REQUESTED** [review]
Agreed, please update `README` accordingly

**Comment by cjyetman** [comment]
verified that this works once #62 is included

File: README.md [hunk 1]
```diff
@@ -10,14 +10,17 @@ The goal of `workflow.prepare.pacta.indices` is to run indices through PACTA,
 and format them for the Transition Monitor webtool. 
 
 ## Running Prepare PACTA Indices workflow  
+
 ### Required input
 
-The index preparation Dockerfile uses the `transitionmonitordockerregistry/rmi_pacta` docker image as a base image. Pulling this image requires access to the Azure docker registry `transitionmonitordockerregistry`. 
+The index preparation Dockerfile uses the `ghcr.io/rmi-pacta/workflow.transition.monitor` docker image as a base image.
+Although the image is public, pullang public images ffrom GHCR requires authentication.
```
Comment (cjyetman) [inline README.md:17]:
```suggestion
Although the image is public, pulling public images from GHCR requires authentication.
```

**Review by cjyetman — CHANGES_REQUESTED** [review]
see minor spelling suggestions

**Review by cjyetman — APPROVED** [review]

**Comment by AlexAxthelm** [comment]
Note, we can't get around the install requirement entirely, since `pacta.data.scraping` needs to be installed, but here's the relevant output from a build I just started
```
#8 55.06
#8 55.11 → Will install 14 packages.
#8 55.26 → All 15 packages (0 B) are cached.
#8 55.28 + askpass               1.2.0
#8 55.28 + curl                  5.1.0       + ✔ libcurl4-openssl-dev, ✔ libssl-dev
#8 55.28 + httr                  1.4.7
#8 55.28 + httr2                 0.2.3
#8 55.28 + logger                0.2.2
#8 55.28 + lubridate             1.9.3
#8 55.28 + openssl               2.1.1       + ✔ libssl-dev
#8 55.28 + pacta.data.scraping   0.1.0.9000 [bld][cmp] (GitHub: 91302af)
#8 55.28 + rvest                 1.0.3
#8 55.28 + selectr               0.4-2
#8 55.28 + sys                   3.4.2
#8 55.28 + timechange            0.2.0
#8 55.28 + V8                    4.4.0       + ✖ libnode-dev
#8 55.28 + xml2                  1.3.5       + ✖ libxml2-dev
#8 55.35 → Will install 2 system packages:
#8 55.37 + libnode-dev  - V8
#8 55.37 + libxml2-dev  - xml2
```

and we can see that. it's not trying to reinstall any of the other pacta packages.

**Comment by AlexAxthelm** [comment]
Would it be prudent to fail the build if (pacta) dependencies are out of date?

**Comment by cjyetman** [comment]
pacta.data.scraping is fine to install, it's just the packages already in the TM image that we should avoid overwriting.

File: Dockerfile [hunk 2]
```diff
@@ -7,9 +7,8 @@
 # https://packagemanager.rstudio.com/client/#/repos/2/overview
 # https://packagemanager.rstudio.com/cran/__linux__/jammy/2023-03-31+MbiAEzHt
```
Comment (cjyetman) [inline Dockerfile:8]: remove these comments if they're not being used

**Review by cjyetman — APPROVED** [review]

**Review by jdhoffa — APPROVED** [review]
I am trusting @cjyetman's review and rubber stamping.
I did not review this myself.

**Pull request merged**